### PR TITLE
docs: update with latest create-mud, useEntityQuery usages

### PR DESCRIPTION
**Pull request by holic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ MUD is MIT-licensed, open source and free to use.
 ## Quickstart
 
 ```
-npx @latticexyz/cli create my-project
+yarn create mud my-project
```
Comment (alvrs) [inline README.md:60]: 😍

File: tutorials/emojimon/step-11.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ Since MUD handles all the networking for us, and because we're using a common ba
 
 We're currently only rendering our own player. To make the game multiplayer, we just need to query for other players and render them too!
 
-```tsx !#1-2,5,9,16-26,37-39,53-58 packages/client/src/GameBoard.tsx
-import { useEffect, useMemo, useState } from "react";
+```tsx #2,5,9,16-24,33,47-52 packages/client/src/GameBoard.tsx
```
Comment (alvrs) [inline tutorials/emojimon/step-11.md:15]: Do we need the `!` here? I forget if `!` hides or shows the line numbers

File: tutorials/emojimon/step-11.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ Since MUD handles all the networking for us, and because we're using a common ba
...
+import { useEffect, useState } from "react";
 import { EntityID, getComponentValueStrict, Has } from "@latticexyz/recs";
-…
+// …
```
Comment (alvrs) [inline tutorials/emojimon/step-11.md:18]: looks like on other pages we just have `...`, should we remove `//` here too for consistency?

File: tutorials/emojimon/step-11.md (hunk 2 above)
Comment (holic) [reply tutorials/emojimon/step-11.md:15]: oh yep, good catch

File: tutorials/emojimon/step-11.md (hunk 3 above)
Comment (holic) [reply tutorials/emojimon/step-11.md:18]: oops, this was also for testing 🙈

**Review by alvrs — APPROVED** [review]

**Pull request merged**